man/rules/meson.build

```diff
@@ -898,6 +898,7 @@ manpages = [
   ''],
```

Comment:
something is wrong in your git setup. you are using your name where you should use your email address, see the git author info for this commit